Lib/test/test_ipaddress.py

```diff
@@ -266,6 +287,41 @@ def test_weakref(self):
 class AddressTestCase_v6(BaseTestCase, CommonTestMixin_v6):
     factory = ipaddress.IPv6Address
 
+    def test_format(self):
+        addr = ipaddress.IPv6Address("::1.2.3.4")
+        pairs  = [
+            ("b"
+            ,"000000000000000000000000000000000"
+            "000000000000000000000000000000000"
+            "00000000000000000000000000000000"
+            "000001000000100000001100000100"),
+            ("n" ,"00000000000000000000000001020304"),
+            ("x" ,"00000000000000000000000001020304"),
+            ("_b"
+            ,"0000_0000_0000_0000_0000_0000_0000_0000_"
+            "0000_0000_0000_0000_0000_0000_0000_0000_"
+            "0000_0000_0000_0000_0000_0000_0000_0000_"
+            "0000_0001_0000_0010_0000_0011_0000_0100"),
+            ("_n" ,"0000_0000_0000_0000_0000_0000_0102_0304"),
+            ("_x" ,"0000_0000_0000_0000_0000_0000_0102_0304"),
+            ("#b"
+            ,"0b000000000000000000000000000000000"
+            "000000000000000000000000000000000"
+            "000000000000000000000000000000000"
+            "00001000000100000001100000100"),
+            ("#n" ,"0x00000000000000000000000001020304"),
+            ("#x" ,"0x00000000000000000000000001020304"),
+            ("#_b"
+            ,"0b0000_0000_0000_0000_0000_0000_0000_0000_"
+            "0000_0000_0000_0000_0000_0000_0000_0000_"
+            "0000_0000_0000_0000_0000_0000_0000_0000_"
+            "0000_0001_0000_0010_0000_0011_0000_0100"),
+            ("#_n" ,"0x0000_0000_0000_0000_0000_0000_0102_0304"),
+            ("#_x" ,"0x0000_0000_0000_0000_0000_0000_0102_0304"),
+        ]
+        for (fmt, txt) in pairs:
+            res = format(addr, fmt)
```

Comment:
Missing an assertion here?

Reply:
Fixed - remove the whole 'res = ...' line.  Cut and paste leftover.